package/com.unity.formats.usd/Tests/Common/Data/UsdImportTests/UsdInstanceVertexTest.usda

```diff
@@ -0,0 +1,42 @@
+#usda 1.0
+(
+    doc = "Blender v3.3.1"
+    metersPerUnit = 1
+    upAxis = "Y"
+)
+
+def Xform "zAxisTest_Instanced"
+{
+    def PointInstancer "instancers" (
+        kind = "group"
+    )
+    {
+        int64[] invisibleIds = []
+        point3f[] positions = [(0, 0, 0)]
+        int[] protoIndices = [0]
+        rel prototypes = </zAxisTest_Instanced/instancers/uniquePointsMesh>
+        uniform token orientation = "leftHanded"
```

Comment:
Can we have additional test cases that tests what happens when this is rightHanded and when the upAxis is different?